test/StartupPerf/startupPerf.ps1

```diff
@@ -0,0 +1,47 @@
+# execute this first: Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process -Confirm 
+
+function GetUrlStatusCode($url)
+{
+	$req = [system.Net.WebRequest]::Create($url)
+	try {
+		$res = $req.GetResponse()
+	} catch [System.Net.WebException] {
+		$res = $_.Exception.Response
+	}
+	return [int]$res.StatusCode
+}
+
+$totalTime = 0
+$attempts = 10
+
+function StartUnderDebugger()
+{
+	$script:sw = [system.diagnostics.stopwatch]::StartNew()
+
+	$dte.ExecuteCommand("Debug.Start");
```

Comment:
To measure startup overhead, shouldn't the `Debug.Start` and `Debug.StopDebugging` commands be executed inside the loop? Right now, only the first GetUrlStatusCode call will incur the startup overhead, while the remaining 200 will incur only the runtime overhead.